polls/views.py

```diff
@@ -54,6 +54,9 @@ def vote(request, question_id):
             'error_message': _("You didn't select a choice."),
         })
     else:
-        selected_choice.update(votes=F('votes') + 1)
+        selected_choice.update(vote_count=F('vote_count') + 1)
+        PollVote.objects.create(user=request.user,
```

Comment:
I would use a `get_or_create` here to make sure a user cannot vote twice. You should be able to figure this out with a test.